packages/client/package.json

```diff
@@ -9,8 +9,10 @@
     "react-dom": "^16.12.0",
     "react-facebook-pixel": "^0.1.3",
     "react-ga": "^2.7.0",
+    "react-geosuggest": "^2.13.0",
```

Comment:
How many KB's does this package contribute?

Reply:
According to react-geosuggest dist folder, the minified version is about 17kb.
I've also checked the package on [bundlephobia](https://bundlephobia.com/result?p=react-geosuggest@2.13.0) and it seems that tree shaking support is availbale when using webpack.